bindata/network/ovn-kubernetes/006-ovs-node.yaml

```diff
@@ -42,6 +42,7 @@ spec:
         - |
           #!/bin/bash
           set -euo pipefail
+          set -x
```

Comment:
@mccv1r0 let's remove this for the final one

Reply:
Agreed